codespell_lib/data/dictionary.txt

```diff
@@ -2102,6 +2124,7 @@ canadan->canadian
 canbe->can be
 cancelability->cancellability
 cancelaltion->cancellation
+cancelled->canceled
```

Comment:
FYI @Fourchaux , this is valid one side of the pond:
https://www.grammarly.com/blog/canceled-vs-cancelled/

Removed in https://github.com/codespell-project/codespell/pull/877 .